pkg/apis/autoscaling/v1alpha1/pa_validation.go

```diff
@@ -34,7 +34,10 @@ func (pa *PodAutoscalerSpec) Validate(ctx context.Context) *apis.FieldError {
 	if equality.Semantic.DeepEqual(pa, &PodAutoscalerSpec{}) {
 		return apis.ErrMissingField(apis.CurrentField)
 	}
-	return serving.ValidateNamespacedObjectReference(&pa.ScaleTargetRef).ViaField("scaleTargetRef").Also(serving.ValidateContainerConcurrency(&pa.ContainerConcurrency).ViaField("containerConcurrency")).Also(validateSKSFields(ctx, pa))
+	return serving.ValidateNamespacedObjectReference(&pa.ScaleTargetRef).
+		ViaField("scaleTargetRef").Also(
+		serving.ValidateContainerConcurrency(ctx, &pa.ContainerConcurrency).
+			ViaField("containerConcurrency")).Also(validateSKSFields(ctx, pa))
```

Comment:
Generally when I wrap these I try to break at the `.Also(`:

```
return foo(bar).ViaField("baz").Also(
     foo(bar).ViaField("baz")).Also(
     foo(bar).ViaField("baz"))

Reply:
Still too long, but I've reformatted the part.